src/Bicep.Core/Semantics/DeclaredSymbol.cs

```diff
@@ -5,14 +5,14 @@
 
 namespace Bicep.Core.Semantics
 {
-    public abstract class DeclaredSymbol : Symbol
+    public abstract class DeclaredSymbol : BindableSymbol
```

Comment:
Out of interest, what's the distinction between `BindableSymbol` & `DeclaredSymbol`?

Reply:
`DeclaredSymbol` requires an `ISymbolContext` which is specific to Bicep files, so `BindableSymbol` is a more general wrapper that works for Bicep param files!